contracts/ArrowVestingFactory.sol

```diff
@@ -23,9 +24,8 @@ contract ArrowVestingFactory {
     }
 
     /**
-        Creates a new cloned vesting wallet from its base implementation.
-
-        The wallet will initially be empty and should have appropriate ERC20 tokens and ETH transferred to it after creation.
+        @notice Creates a new cloned vesting wallet from its base implementation
+        @dev The wallet will initially be empty and should have appropriate ERC20 tokens and ETH transferred to it after creation
```

Comment:
Is this a notice of developers? It seems useful to know for all users of the contract.